parlai/core/dict.py

```diff
@@ -248,9 +248,6 @@ def __init__(self, opt: Opt, shared=None):
             'dict_textfields', DictionaryAgent.default_textfields
         ).split(",")
 
-        # used to signal whether we should use training time tricks, like bpe droput
-        self._tokenization_mode = TokenizationMode.TEST_TIME_LABEL
```

Comment:
I believe this was dead code.

Reply:
not sure? Dictionary should be initialized into evaluation mode by default. Maybe it should be TEST_TIME_TEXT?

Anyway if it's initializing to never dropping out, it's ok